packages/app/prisma-utils/src/prismaClientBuilder.ts

```diff
@@ -0,0 +1,16 @@
+import { PrismaClient } from '@prisma/client'
+import type { PrismaClientBuilderOptions } from './types'
+
+const defaultOptions: PrismaClientBuilderOptions = {
+  transactionOptions: { isolationLevel: 'ReadCommitted' },
+}
+
+export const prismaClientBuilder = (options: PrismaClientBuilderOptions = {}): PrismaClient => {
```

Comment:
builder pattern assumes preparing configuration via fluent API and then invoking build(), having it as just a function might be confusing.
maybe we can call it a factory?

Reply:
You are absolutely right! I will address the change, thanks Igor!

Reply:
Change applied!